app/src/main/res/values/colors.xml

```diff
@@ -10,5 +10,7 @@
     <color name="colorTitle">#EBEBEB</color>
     <color name="colorText">#F7F6F6</color>
     <color name="colorDescription">#C0C0C0</color>
-    <color name="toolbar">#db4c18</color>
+    <color name="toolbar">#DB4C18</color>
```

Comment:
WARNING: [UnusedResources] The resource `R.color.toolbar` appears to be unused
 Unused resources make applications larger and slow down builds.